LOL, the tests are failing again.
```
FAILED tests/components/light/test_device_trigger.py::test_if_fires_on_state_change_with_for[pyloop]
```
I guess I'll never get my PR merged ;-)

--- a/homeassistant/components/mobile_app/webhook.py
+++ b/homeassistant/components/mobile_app/webhook.py
@@ -184,8 +184,10 @@ async def handle_webhook(
         _LOGGER.error("Received invalid webhook type: %s", webhook_type)
         return empty_okay_response()
 
+    device_name = config_entry.data[ATTR_DEVICE_NAME]
+
     _LOGGER.debug(
-        "Received webhook payload for type %s: %s", webhook_type, webhook_payload
+        "Received webhook payload from %s for type %s: %s", device_name, webhook_type, webhook_payload
     )
 
     # Shield so we make sure we finish the webhook, even if sender hangs up.

--- a/homeassistant/components/mobile_app/webhook.py
+++ b/homeassistant/components/mobile_app/webhook.py
@@ -187,7 +187,10 @@ async def handle_webhook(
     device_name = config_entry.data[ATTR_DEVICE_NAME]
 
     _LOGGER.debug(
-        "Received webhook payload from %s for type %s: %s", device_name, webhook_type, webhook_payload
+        "Received webhook payload from %s for type %s: %s",
+        device_name,
+        webhook_type,
+        webhook_payload,
     )
 
     # Shield so we make sure we finish the webhook, even if sender hangs up.

